samza-shell/src/main/bash/run-class.sh

```diff
@@ -39,8 +39,8 @@ fi
 HADOOP_YARN_HOME="${HADOOP_YARN_HOME:-$HOME/.samza}"
 HADOOP_CONF_DIR="${HADOOP_CONF_DIR:-$HADOOP_YARN_HOME/conf}"
 GC_LOG_ROTATION_OPTS="-XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=10241024"
-LOG4J_FILE_NAME="log4j.xml"
-LOG4J2_FILE_NAME="log4j2.xml"
+[[ -z ${LOG4J_FILE_NAME+x} ]] && LOG4J_FILE_NAME="log4j.xml"
```

Comment:
can be simplified as
`LOG4J_FILE_NAME=${LOG4J_FILE_NAME:=log4j.xml}`

Reply:
Love this!